Makefile

```diff
@@ -95,12 +101,27 @@ set-docs-owner:
 
 archive:
 	@git archive --format=tar --prefix=$(PKGNAME)-$(VERSION)/ $(TAG) > $(PKGNAME)-$(VERSION).tar
-	@gzip $(PKGNAME)-$(VERSION).tar
+	@gzip -f $(PKGNAME)-$(VERSION).tar
 	@echo "The archive is in $(PKGNAME)-$(VERSION).tar.gz"
 
 dist: tag archive
 	scp $(PKGNAME)-$(VERSION).tar.gz fedorahosted.org:lorax
 
+rpm: archive $(PKGNAME).spec
```

Comment:
Is that still an actually useful target? Or just for developer convenience? If the former, please add it to `.PHONY`.